src/main/java/taskutil/Task.java

```diff
@@ -0,0 +1,71 @@
+package taskutil;
+
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Class for Task objects and methods to modify.
+ */
+public class Task {
+
+    // Format of date to be displayed to user.
+    protected static final DateTimeFormatter DISPLAY_FORMAT = DateTimeFormatter.ofPattern("dd MMM yyyy, hh:mm a");
+
+    protected String title;
+    protected Boolean isDone;
+
+    /**
+     * Constructor for general Task object, called in subclasses.
+     *
+     * @param title Description of task.
+     */
+    public Task(String title) {
+        this.title = title;
+        this.isDone = false;
+    }
+
+    /**
+     * Gets completed status of task.
+     *
+     * @return X if task is completed, a blank space otherwise.
+     */
+    public String getStatus() {
+        return (isDone ? "X" : " ");
+    }
+
+    /**
+     * Modifies status of current task.
+     *
+     * @param isCompleted New status of task.
+     */
+    public void changeStatus(boolean isCompleted) {
```

Comment:
maybe a bit of a nitpick, but should it be setStatus? (since it could be the same as the current status, e.g)